modules/jaeger-config-storage.adoc

```diff
@@ -189,18 +189,18 @@ spec:
 [id="jaeger-config-external-es_{context}"]
 == Connecting to an existing Elasticsearch instance
 
-Jaeger also allows you to use an existing (self-provisioned) Elasticsearch cluster for storage. You do this by specifying the URL of the existing cluster as the `spec:storage:options:es:server-urls` value in your configuration.
+Jaeger also allows you to use an already existing Elasticsearch cluster for storage (that is, an instance that was not auto-provisioned by the Jaeger Operator calling the Red Hat Elasticsearch Operator). You do this by specifying the URL of the existing cluster as the `spec:storage:options:es:server-urls` value in your configuration.
 
 .Restrictions
 
-* There can be only one Jaeger with self-provisioned Elasticsearch instance per namespace.
+* You cannot share or reuse a {ProductName} logging Elasticsearch instance with Jaeger. The Elasticsearch cluster is meant to be dedicated for a single Jaeger instance.
 
 [NOTE]
 ====
 Red Hat does not provide support for your external Elasticsearch instance. You can review the tested integrations matrix on the link:https://access.redhat.com/articles/5381021[Customer Portal].
 ====
 
-The following configuration parameters are for an _external_ Elasticsearch instance, that is, an instance that was not created using the OpenShift Elasticsearch Operator. You specify configuration options for external Elasticsearch under `spec:storage:options:es` in your configuration file.
+The following configuration parameters are for an already existing Elasticsearch instance (also known as an _external_  Elasticsearch instance). In this case, you specify configuration options for Elasticsearch under `spec:storage:options:es` in your configuration file.
```

Comment:
I guess I'm missing the bigger context here, but what's the configuration file? Is this the CR?